geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DiskStoreCommands.java

```diff
@@ -639,29 +639,34 @@ public Result compactOfflineDiskStore(
       String errorString = CliStrings.format(
           CliStrings.COMPACT_OFFLINE_DISK_STORE__MSG__ERROR_WHILE_COMPACTING_DISKSTORE_0_WITH_1_REASON_2,
           diskStoreName, fieldsMessage);
-      result = ResultBuilder.createUserErrorResult(errorString);
+      result = ResultBuilder.createUserErrorResult(CliStrings.format(
```

Comment:
This looks to me like it's going to build a strange error message that I expect will look something like this:: 

```
An error occurred while doing compaction: "While compacting disk store={0} {1}. Reason: {2}"
``` 

I think {0} and {1} will be filled in but not {2}.